zhaquirks/terncy/cl001.py

```diff
@@ -0,0 +1,88 @@
+"""Quirk for Xiaoyan CL001 ceiling light."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class CustomColorCluster(CustomCluster, Color):
+    """Set actual supported CCT range and remove RGB color picker since hardware does not support it."""
+
+    _CONSTANT_ATTRIBUTES = {
+        Color.AttributeDefs.color_capabilities.id: Color.ColorCapabilities.Color_temperature,
+        Color.AttributeDefs.color_temp_physical_min.id: 50,
+        Color.AttributeDefs.color_temp_physical_max.id: 500,
+    }
+
+
+class LuminousElement(CustomDevice):
+    """System call, generate luminous element. Adhere."""
```

Comment:
```suggestion
class TerncyLightCCT(CustomDevice):
    """Terncy Light CCT device."""
```

Reply:
@rts-kotori Can you merge this last change, so the PR can be merged too? Thanks!